### ISSUE TITLE
Pausing does not invalidate cache for some apps

### ISSUE THREAD

**Issue by ChaosNicro** [IngoZenz/personaldnsfilter#261]
Heya
This is more a nag than a problem so absolutely low priority. 
When I hit a blocked entry in Firefox and disable the filter to temporarily get through, reloading the page still shows it blocked. Only after a couple minutes does it update to load the page on a refresh. Pausing before the first load or switching browsers avoids the issue. I can also clear the cache from the app-info of the browser.

I noticed that other DNS-Apps disengage the vpn-interface when paused, which seems to invalidate the cached requests. Though since only rhe filter component is paused, that might not be an option here.

**Comment by ChaosNicro** [comment]
Turns out you can skip the cache with a gesture in-browser. This does enough to solve my immediate issue.
